uaclient/api/u/pro/security/status/reboot_required/v1.py

```diff
@@ -1,29 +1,111 @@
+from typing import List, Optional
+
 from uaclient.api.api import APIEndpoint
 from uaclient.api.data_types import AdditionalInfo
 from uaclient.config import UAConfig
-from uaclient.data_types import DataObject, Field, StringDataValue
-from uaclient.security_status import get_reboot_status
+from uaclient.data_types import (
+    BoolDataValue,
+    DataObject,
+    Field,
+    StringDataValue,
+    data_list,
+)
+from uaclient.livepatch import status
```

Comment:
I suggest 
```suggestion
from uaclient import livepatch
```

because then later it would be `livepatch.status()` which is clear. When it's just `status` it looks like it could be the normal overall pro status function.